libcnb-test/src/build.rs

```diff
@@ -0,0 +1,65 @@
+use fs_extra::dir::CopyOptions;
+use libcnb_cargo::cross_compile::{cross_compile_assistance, CrossCompileAssistance};
+use libcnb_cargo::{
+    assemble_buildpack_directory, build_buildpack_binary, BuildError, CargoProfile,
+};
+use std::path::{Path, PathBuf};
+use tempfile::{tempdir, TempDir};
+
+/// Packages the current crate as a buildpack into a temporary directory.
+pub(crate) fn package_crate_buildpack(
+    target_triple: impl AsRef<str>,
+    app_dir: impl AsRef<Path>,
+) -> Result<TempDir, PackageCrateBuildpackError> {
+    let cargo_manifest_dir = std::env::var("CARGO_MANIFEST_DIR")
+        .map(PathBuf::from)
+        .map_err(PackageCrateBuildpackError::CannotDetermineCrateDirectory)?;
+
+    let temp_app_dir =
+        tempdir().map_err(PackageCrateBuildpackError::CannotCreateAppTempDirectory)?;
+
+    let mut copy_options = CopyOptions::new();
+    copy_options.content_only = true;
+
+    fs_extra::dir::copy(
+        &cargo_manifest_dir.join(app_dir),
+        temp_app_dir.path(),
+        &copy_options,
+    )
+    .map_err(PackageCrateBuildpackError::CannotCopyAppToTempDirectory)?;
```

Comment:
Is this a leftover? It seems to duplicate the `app::copy_app` call made in `IntegrationTest::run`?

Reply:
Wow, great catch. Yeah - leftover from refactoring. Thanks! 👍🏻